src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenOperators.cs

```diff
@@ -2277,26 +2277,14 @@ static int Main()
         int errCount = 0;
         var src = new X();
         // QE is not 'executed'
-        var b = false && from x in src select x; // WRN CS0429
+        var b = false && from x in src select x;
         if (src.selectCount == 1)
             errCount++;
 
         Console.Write(errCount);
         return (errCount > 0) ? 1 : 0;
     }
 }";
-            // the grammar does not allow a query on the right-hand-side of &&, but we allow it except in strict mode.
-            CreateCompilationWithMscorlib40AndSystemCore(source, parseOptions: TestOptions.Regular.WithStrictFeature()).VerifyDiagnostics(
```

Comment:
>CreateCompilationWithMscorlib40AndSystemCore [](start = 12, length = 44)

We should continue to test this case.

Reply:
I'm not sure what you mean.  Strict more no longer diagnoses this.  Are you suggesting that we should show the warning-wave warnings in this test?

---
In reply to: [463293894](https://github.com/dotnet/roslyn/pull/46239#discussion_r463293894) [](ancestors = 463293894)

Reply:
> Are you suggesting that we should show the warning-wave warnings in this test?

Yes, sorry, that wasn't clear in my earlier statement.

---
In reply to: [463310390](https://github.com/dotnet/roslyn/pull/46239#discussion_r463310390) [](ancestors = 463310390,463293894)